src/pykeen/models/base.py

```diff
@@ -616,8 +616,8 @@ def __init__(
         else:
             self.regularizer = NoRegularizer()
 
-        self._entity_ids = triples_factory.entity_ids
-        self._relation_ids = triples_factory.relation_ids
+        self._entity_ids = set(range(triples_factory.num_entities))
```

Comment:
could be a function/property of the DatasetInfo object?

Reply:
I guess the better way would be to get rid of this attribute; it only seems to be used in extend_batch, where it could actually be `torch.arange(num_entities)` instead.

Reply:
iirc, it originated from a time where we where not sure whether we should support non-consecutive indices - now we enforce this, so this _should_ not be of relevance anymore.

Reply:
a45079d...b7621a1